cmd/contour/serve.go

```diff
@@ -1209,6 +1211,7 @@ func (s *Server) getDAGBuilder(dbc dagBuilderConfig) *dag.Builder {
 			GlobalRateLimitService:        dbc.globalRateLimitService,
 			PerConnectionBufferLimitBytes: dbc.perConnectionBufferLimitBytes,
 			SetSourceMetadataOnRoutes:     true,
+			GlobalCircuitBreakerDefaults:  dbc.globalCircuitBreakerDefaults,
```

Comment:
I think it makes sense to have these global defaults take effect for all three processors (Ingress, HTTPProxy, Gateway) -- so need to add the implementation to `internal/dag/gatewayapi_processor.go`, and then make sure we're passing the defaults into all three processors here when we instantiate them.

Reply:
Yeah I did for `ingress` and `httpproxy`, I wasnt aware if there was anything specific to gateway api spec that made this feature not that useful. Will add it